packages/tenant-process/src/services/tenantService.ts

```diff
@@ -648,6 +649,82 @@ export function tenantServiceBuilder(
       await repository.createEvent(event);
     },
 
+    async internalRevokeCertifiedAttribute(
```

Comment:
Could this share some logic with #421?

Reply:
I added the `revokeCertifiedAttribute` function to #421  to reuse the code, but the two branches are not connected, so I rewrote it here too.